src/Components/SectionCarousel/SectionCarousel.jsx

```diff
@@ -0,0 +1,11 @@
+import './SectionCarousel.css';
+
+const SectionCarousel = () => {
+  return (
+    <section className="Slide-Caroussel">
```

Comment:
plz use the camelCase like the other classNames !

Reply:
The problem is resolved.